src/components/calcite-color-picker/readme.md

```diff
@@ -2,6 +2,22 @@
 
 <!-- Auto Generated Below -->
 
+## Usage
+
+### Basic
+
+```html
+<calcite-color-picker dir="ltr" scale="m" value="#b33f33" appearance="default" />
```

Comment:
Can we add the closing tag? The closing tag is required for custom elements. I think this will only work in JSX.